src/toil/batchSystems/slurm.py

```diff
@@ -212,17 +212,7 @@ def run(self):
 
     def prepareSbatch(self, cpu, mem, jobID):
         #  Returns the sbatch command line before the script to run
-        # sbatch on slurm is analogous to qsub on SGE. Below are the qsub args and their meanings
-        # for translation to slurm
-        # -b is binding y - not relevant for slurm
-        # -terse means just return the job id. Can I use -Q?
-        # -j means merge stderr + stdout (not necessary)
-        # -cwd means use the current working directory (not necessary)
-        # -o means standard output, but I think I can redirect this to /dev/null (done
-        # -e means standard error (done)
-        # -N is job name, which should be based on the jobID
-
-        sbatch_line = ['sbatch', '-Q', '-J', 'toil_job_{}'.format(jobID)]
+        sbatch_line = ['sbatch', '-Q', '-J', 'toil_job_{}'.format(jobID), '-o', '/dev/null', '-e', '/dev/null']
```

Comment:
This is what the GridEngine batch system does and turned out to be a bad idea. If anything goes wrong before worker.py gets a chance to setup its own redirection, any errors that occur will be swallowed, making diagnosing  problems much harder. 

We have something in the works that will allow us to set a workflow-specific location for files like this. For the time being, could you undo this change and instead add the appropriate clean-up code to the tests?